packages/flutter/test/material/input_chip_test.dart

```diff
@@ -294,41 +291,42 @@ void main() {
     focusNode2.dispose();
   });
 
-  testWidgets('Material2 - Input chip check mark color is determined by platform brightness when light', (WidgetTester tester) async {
+  testWidgets('Material2 - Input chip disabled check mark color is determined by platform brightness when light', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData(useMaterial3: false);
     await pumpCheckmarkChip(
       tester,
       chip: selectedInputChip(),
-      useMaterial3: false,
+      theme: theme,
     );
 
     expectCheckmarkColor(find.byType(InputChip), Colors.black.withAlpha(0xde));
   });
 
-  testWidgets('Material3 - Input chip check mark color is determined by platform brightness when light', (WidgetTester tester) async {
-    await pumpCheckmarkChip(tester, chip: selectedInputChip());
-
-    expectCheckmarkColor(find.byType(InputChip), Colors.black.withAlpha(0xde));
+  testWidgets('Material3 - Input chip disabled check mark color is determined by platform brightness when light', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData(useMaterial3: true);
+    await pumpCheckmarkChip(tester, chip: selectedInputChip(), theme: theme);
+    expectCheckmarkColor(find.byType(InputChip), theme.colorScheme.onSurface);
   });
 
-  testWidgets('Material2 - Input chip check mark color is determined by platform brightness when dark', (WidgetTester tester) async {
+  testWidgets('Material2 - Input chip disabled check mark color is determined by platform brightness when dark', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData.dark(useMaterial3: false);
     await pumpCheckmarkChip(
       tester,
       chip: selectedInputChip(),
-      brightness: Brightness.dark,
-      useMaterial3: false,
+      theme: theme,
     );
 
     expectCheckmarkColor(find.byType(InputChip), Colors.white.withAlpha(0xde));
   });
 
-  testWidgets('Material3 - Input chip check mark color is determined by platform brightness when dark', (WidgetTester tester) async {
+  testWidgets('Material3 - Input chip disabled check mark color is determined by platform brightness when dark', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData.dark(useMaterial3: true);
```

Comment:
```suggestion
    final ThemeData theme = ThemeData.dark();
```

This checks the theme value but it is one off so we can keep the variable without passing; M3 is true by default.